Connector/utils/utils.py

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/python
+import json
+import os
+from httputils import error, httputils
+from logger import logger
+from .constants import *
+
+
+def getAvailableCurrenciesFile():
```

Comment:
Don't understand this function wrapper. We can call `getConfigProperty` with the `AVAILABLE_CURRENCIES_FILE_PROPERTY` argument directly from code

Reply:
It can be change, but it has no priority